cypress-io/cypress
# fix: upgrades http-proxy to prevent closed socket crash

- Closes https://github.com/cypress-io/cypress/issues/27552

### Additional details
- If the socket is not open the write will fail and crash the server and cypress.

### Steps to test
Reproduction steps are not concrete as this type of issue happens sporadically. This is a temporary validation of the changes introduced in https://github.com/cypress-io/node-http-proxy/pull/1 to confirm they do not break existing proxy functionality and tests.

### How has the user experience changed?
N/A

### PR Tasks


- na Have tests been added/updated?
- na Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- na Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
